lib/litejob/server.rb

```diff
@@ -33,6 +33,18 @@ def pop(queue)
       result
     end
 
+    def kill
+      if @scheduler
+        @schedule.context.kill
+      end
```

Comment:
You have a typo here with `@schedule` instead of `@scheduler`, plus I personally want to avoid one-line conditional nesting, and I'd like to have the return value suggest whether the call was effective or not:
```suggestion
      @scheduler&.context&.kill || false
```

Reply:
Whoops, good point. Updated 👍🏼